core/src/main/resources/reference.conf

```diff
@@ -178,7 +178,7 @@ cassandra-journal {
     # do not change this to Minute. Each tag has in the tag-views table has a partition
     # per tag per bucket
     # Valid options: Day, Hour, Minute
-    bucket-size = "Hour"
+    bucket-size = "Day"
```

Comment:
Perhaps Hour is a better default?

Reply:
Yeah i put this back to make sure there were no surprises when migrating. Also if people leave the default first-bucket results in a lot of queries for events by tag queries

Reply:
the problem is that it can't (easily) be changed afterwards and then people will start out new projects with the defaults, so I think it's more important to have a good default and then document it for migrations. Wasn't the conclusion that Day is probably too big?

Reply:
Let's set it to Hour then :+1: